### PR TITLE
Solve some volume group update and delete errors.

### PR DESCRIPTION
**Pull request by jackhaibo**
**What this PR does / why we need it**:
Solve some volume group update and delete errors.

### CODE REVIEW

**Comment by leonwanghui** [comment]
@wisererik @xxwjj PTAL

File: contrib/drivers/lvm/targets/iscsi.go [hunk 1]
```diff
@@ -86,8 +86,9 @@ func (t *tgtTarget) getTgtConfPath(volId string) string {
 type configMap map[string][]string
 
 func (t *tgtTarget) CreateISCSITarget(volId, tgtIqn, path, hostIp, initiator string, chapAuth []string) error {
-	if hostIp == "" {
-		return errors.New("create ISCSI target failed, host ip cannot be empty")
+	// Multi-attach require a specific ip
+	if hostIp == "" || hostIp == "ALL" {
+		return errors.New("create ISCSI target failed: host ip cannot be empty")
```
Comment (xxwjj) [inline contrib/drivers/lvm/targets/iscsi.go]: the error is not corrct.

Comment (leonwanghui) [reply contrib/drivers/lvm/targets/iscsi.go]: Besides, some log msg may be required here.

Comment (jackhaibo) [reply contrib/drivers/lvm/targets/iscsi.go]: done.

File: contrib/drivers/lvm/targets/iscsi.go [hunk 2]
```diff
@@ -86,8 +87,14 @@ func (t *tgtTarget) getTgtConfPath(volId string) string {
...
-	if hostIp == "" {
-		return errors.New("create ISCSI target failed, host ip cannot be empty")
+	// Multi-attach require a specific ip
+	if hostIp == "" || hostIp == "ALL" {
+		return fmt.Errorf("create ISCSI target failed: host ip %s cannot be empty or ALL, iscsi only allows specific ip access, not all", hostIp)
+	}
+
+	result := net.ParseIP(hostIp)
+	if result == nil {
+		return fmt.Errorf("%s is not a valid ip. Please give the proper ip", hostIp)
```
Comment (leonwanghui) [inline contrib/drivers/lvm/targets/iscsi.go]: %s is not a valid ip, please give the proper ip

Comment (leonwanghui) [reply contrib/drivers/lvm/targets/iscsi.go]: Also require log here.

File: pkg/api/controllers/volumeGroup.go [hunk 3]
```diff
@@ -127,21 +127,38 @@ func (v *VolumeGroupPortal) UpdateVolumeGroup() {
...
 	if err != nil {
 		errMsg := fmt.Sprintf("marshal volume group updated result failed: %s", err.Error())
 		v.ErrorHandle(model.ErrorInternalServer, errMsg)
 		return
 	}
+
 	v.SuccessHandle(StatusAccepted, body)
 
+	// No more values in group need to be updated
+	if addVolumes == nil && removeVolumes == nil {
```
Comment (leonwanghui) [inline pkg/api/controllers/volumeGroup.go]: I prefer using `len(addVolumes) == 0 && len(removeVolumes) == 0` here.

**Pull request merged**